app/pibakery.js

```diff
@@ -70,11 +70,12 @@ var tempBlocks = []
   * @return null
 */
 function getOsPath () {
-  if (process.platform == 'darwin') {
+  var platform = process.platform;
+  if ((platform == 'darwin') || (platform == 'linux') || (platform == 'freebsd')) {
     // Mac stores the OS in Application Support
     return path.normalize('/Library/Application Support/PiBakery/os/')
```

Comment:
Yes, David. Cool, I wasn't familiar of the OSX limitation of 1GB apps.
I have made this change on my fork and will merge the official update from this repo as and when it becomes available.